.gitignore

```diff
@@ -110,3 +110,5 @@ public/
 notes.txt
 .idea/
 .vscode/
+
+react/src/static/slack-app-clone-web/translations/en.json
```

Comment:
its part of build. can this moved to `dist` / `build` directory?

Refer this one - https://github.com/web-slate/slack-app-clone-web/blob/main/react/webpack.config.js#L89